src/ProductConstructionService/ProductConstructionService.Api/AuthenticationConfiguration.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Reflection;
+using Maestro.Authentication;
+using Microsoft.DotNet.GitHub.Authentication;
+
+namespace ProductConstructionService.Api;
+
+public static class AuthenticationConfiguration
+{
+    public const string GitHubAuthenticationKey = "GitHubAuthentication";
+    public const string GitHubClientIdKey = "ClientId";
+    public const string GitHubClientSecretKey = "ClientSecret";
+
+    // We want to use the AuthenticationScheme for all of our calls
+    public const string AuthenticationSchemeRequestPath = "";
```

Comment:
Should this be empty? Maybe we can have a comment what empty means?
Maybe we could also just use `string.Empty`?

Reply:
We want it to be an empty string. I added more comments and changed it to a private readonly so it can be `string.Empty`